utilities/shared_disk.py

```
@@ -253,58 +296,644 @@ def verify_shared_disk_data(
+    except (GuestCommandError, vim.fault.VimFault, TimeoutExpiredError, ConnectionError, OSError) as e:
+        # Best-effort: Fast Startup disable is non-critical — migration can succeed
+        # without it, but may produce ConversionHasWarnings.
+        LOGGER.warning(f"Failed to disable Fast Startup on '{owner_name}' (best-effort): {e}")
+
+
+def _get_guest_auth(
+    source_provider: VMWareProvider,
+    owner_vm: vim.VirtualMachine,
+    owner_name: str,
+    source_provider_data: dict[str, Any],
+) -> tuple["vim.vm.guest.NamePasswordAuthentication", str]:
```

Ruff (0.15.18)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)